### ISSUE TITLE
Uncaught TypeError: t.remove is not a function

### ISSUE THREAD

**Issue by paulcanning** [paperjs/paper.js#1449]
I have an SVG saved to JSON, and I'm trying to import it back into the canvas.

I've tried the following:

`var symbol = new paper.SymbolDefinition(data);`

but I'm getting:

> Uncaught TypeError: t.remove is not a function

How can I import my SVG path?

**Comment by lehni** [comment]
I don't understand what an SVG as JSON means... SVG is its own format, and has nothing to do with JSON.

You can import Paper.js JSON data with http://paperjs.org/reference/project/#importjson-json
And SVG with http://paperjs.org/reference/project/#importsvg-svg

Nowhere in the documentation it says that `SymbolDefinition` can be used to import foreign data... So this is isn't a bug. 

Support questions are better directed at the mailing list: http://groups.google.com/group/paperjs

**Comment by paulcanning** [comment]
I'm loading an SVG file in (paths contained in an actual file format) to the canvas. I then want to to save that data (paths) to a JSON string, so it can be stored in a database, as JSON, rather than saving the filename of the SVG file.

**Comment by lehni** [comment]
This should help with exporting: http://paperjs.org/reference/project/#exportsvg

**Comment by paulcanning** [comment]
But I already exported it! I simply applied JSON.stringify() to my SVG object (after it was imported to the canvas, which is where I got the JSON object.

I'm trying to _import_ the SVG JSON Object, but I can't see how.

The SVG **file** is added using importSVG and making a symboldefinition, but using a symboldefinition doesn't work for the JSON object representation of the SVG (see opening post)
